spec/models/manageiq/providers/cloud_manager/provision_workflow_spec.rb

```diff
@@ -0,0 +1,181 @@
+describe ManageIQ::Providers::CloudManager::ProvisionWorkflow do
+  include Spec::Support::WorkflowHelper
+
+  let(:admin) { FactoryGirl.create(:user_with_group) }
+  let(:ems) { FactoryGirl.create(:ems_amazon) }
+  let(:template) { FactoryGirl.create(:template_amazon, :name => "template", :ext_management_system => ems) }
```

Comment:
Can we use generic factories where possible.  Example: `template_cloud`.

This we need a little more work before we can replace all of them.